### DISCUSSION TITLE
Failed to resolve import "@/service/ProductService"

### DISCUSSION THREAD

**Discussion by abhishek-junghare** [primefaces/community#3133] [PrimeVue]
I am a beginner, and I don't know what this is. There is no `service` folder inside my directory.

**Comment by j0rgedev** [comment]
Hello @abhishek-junghare. Don't worry, we all started there! I imagine you're trying to replicate some example code from some component. What you need to do, is go to the stackblitz reproduction, which is a lightning icon, and look at the ProductService.js and just copy the code into your project
